src/app/App.js

```diff
@@ -2,47 +2,72 @@ import { Component } from 'react';
 import Header from './Header';
 import Footer from './Footer';
 import Home from '../home/Home';
+import Auth from '../auth/Auth';
 import {
   BrowserRouter as Router,
   Route,
   Switch,
   Redirect
 } from 'react-router-dom';
 import './App.css';
+import TodoList from '../todos/TodosList';
 
 class App extends Component {
 
+  state = {
+    token: (window.localStorage.getItem('TOKEN')) ? window.localStorage.getItem('TOKEN') : '',
```

Comment:
if you wanted to, you could set window.location.getItem("TOKEN") as a variable (I think a function would be better) and make this read a little better. 
if you wanted to go the function route:
```
export const userToken = () => {
  return (window.localStorage.getItem('TOKEN'))
        ? window.localStorage.getItem('TOKEN')
        :  ''
}
```
I think that should work, or at least be close to working. 
Now you have a function you can pass anywhere to find a token